ui/components/multichain/network-list-menu/popular-network-list/popular-network-list.tsx

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import { ApprovalType } from '@metamask/controller-utils';
+import { useHistory } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import {
+  Box,
+  Text,
+  AvatarNetwork,
+  Button,
+  AvatarNetworkSize,
+  ButtonVariant,
+  ButtonPrimary,
+  ButtonPrimarySize,
+} from '../../../component-library';
+import { MetaMetricsNetworkEventSource } from '../../../../../shared/constants/metametrics';
+import {
+  ENVIRONMENT_TYPE_POPUP,
+  ORIGIN_METAMASK,
+} from '../../../../../shared/constants/app';
+import {
+  requestUserApproval,
+  toggleNetworkMenu,
+} from '../../../../store/actions';
+import { ADD_NETWORK_ROUTE } from '../../../../helpers/constants/routes';
+import { getEnvironmentType } from '../../../../../app/scripts/lib/util';
+import {
+  AlignItems,
+  BackgroundColor,
+  BlockSize,
+  BorderRadius,
+  Display,
+  FlexDirection,
+  JustifyContent,
+  TextAlign,
+  TextColor,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { RPCDefinition } from '../../../../../shared/constants/network';
+
+const PopularNetworkList = ({
+  searchAddNetworkResults,
+}: {
+  searchAddNetworkResults: RPCDefinition[];
+}) => {
+  const t = useI18nContext();
+  const isPopUp = getEnvironmentType() === ENVIRONMENT_TYPE_POPUP;
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  if (Object.keys(searchAddNetworkResults).length === 0) {
```

Comment:
If the user has already added all popular networks, can we just not show that list?